src/components/MenuItem.js

```diff
@@ -150,7 +150,7 @@ function MenuItem(props) {
                                             isHovered={isHovered}
                                             isPressed={pressed}
                                             icons={props.icon}
-                                            size={CONST.AVATAR_SIZE.DEFAULT}
+                                            size={props.avatarSize || CONST.AVATAR_SIZE.DEFAULT}
```

Comment:
Let's set the default value of `avatarSize` size to `CONST.AVATAR_SIZE.DEFAULT` and remove this kind of fallback.